.changeset/blue-pans-suffer.md

```diff
@@ -2,7 +2,14 @@
 "@asyncapi/generator": minor
 ---
 
-- **Addition of new config file `.ageneratorrc`**: Previously, generator configuration had to be defined in the `package.json` file. Now, you can define the configuration in a separate `.ageneratorrc` file. The configuration defined in the `.ageneratorrc` file will override any configuration defined in `package.json`. The generator will first check for the `.ageneratorrc` file in the template's root directory, and if not found, it will look for the generator config in `package.json`.
+### What's New
+
+- Introduced a new `conditionalGeneration` configuration for templates.
+- Supports conditional generation of both files and folders.
+- Enables conditions based on values from the AsyncAPI document (`subject`) and custom template `parameters`.
+- The **`parameter`** allows users to pass custom flags or values at generation time (via the CLI or programmatically) to control what parts of the template get rendered.
+-  **Updating `.ageneratorrc` with more configuration** to sync the configuration changes due to conditionalGeneration property in template.
```

Comment:
since now we have one changeset for 2 features, it doesn't make sense to write about `Updating `.ageneratorrc` as it was not release before this PR yet anyway

so just remove this bulletpoint

Reply:
Removed thanks : )